docs/modules/migrate/pages/data-migration-tool.adoc

```diff
@@ -1,65 +1,124 @@
 = Using the Data Migration Tool
-:description: This guide explains how to migrate data from Hazelcast 4.x or 5.x clusters to newer 5.x clusters.
+:description: You can use the Data Migration Tool (DMT) to migrate your data from version 4.x or 5.x Open Source and Enterprise Edition Hazelcast clusters when upgrading to 5.3.x versions of Enterprise Edition, or moving to the latest {hazelcast-cloud} release.
 
-[blue]*Hazelcast Enterprise Feature*
+{description} 
 
-{description} The purpose of Data Migration Tool (DMT) is to migrate data from your Open Source or Enterprise clusters, to either a newer Enterprise cluster or a cluster on Hazelcast Viridian.
+NOTE: The DMT migrates your data for maps and replicated maps only. Replicated map metadata is not migrated.
 
-Currently, the Data Migration Tool (DMT) supports:
+The DMT can be run on Mac, Linux, and Windows Operating Systems.
 
-* migrating data only for maps and replicated maps
-* migrating data from Hazelcast 4.x.y and 5.x.y clusters to Hazelcast 5.3.2 and newer clusters.
+== Get the DMT
```

Comment:
Before we go into downloading and using DMT we should describe the typical migration procedure from high-level point of view:


Initial state: The source cluster is running, it has all the data, and the applications are using the source cluster. The target cluster is setup, it has no data.

1. setup migration cluster
1. shutdown the applications
1. execute the migration using the dmt command
1. start the applications with the new client configuration pointing to the target cluster

The important bit here is that the applications are shut down for the duration of the migration.

Reply:
Added to intro

Reply:
Also added the additional shutdown/restart application steps to the Migrate Your Data section